webcompat/templates/dashboard/triage/stats.html

```diff
@@ -62,7 +84,7 @@ <h2 class="wc-Stat-title">Needstriage issues</h2>
     </div>
   </article>
   <article class="wc-Stats-article">
-    <div class="wc-Stat">
+    <div class="wc-Stat {% if stats.total > 50 %} wc-Stat--alert {% endif %}">
```

Comment:
@magsout Maybe not needed here indeed. This number is always lower or equal to the total. It's a subproduct of the total.